shanoir-ng-front/src/app/carmin/models/parameter-resources-d-t.o.ts

```diff
@@ -0,0 +1,10 @@
+import {GroupByEnum} from "./groupby.enum";
+
+export class ParameterResourcesDTO {
```

Comment:
Why this file name ? Generally DTO is defined internally in the model file I think